client/extensions/woocommerce/app/settings/payments/payments-off-site.js

```diff
@@ -58,12 +62,29 @@ class SettingsPaymentsOffSite extends Component {
 							<ListItemField className="payments__methods-column-settings">
 							</ListItemField>
 						</ListHeader>
-						{ methods && methods.map( this.renderMethodItem ) }
+						{ paymentMethods && paymentMethods.map( this.renderMethodItem ) }
 					</List>
 			</div>
 		);
 	}
+}
+
+function mapStateToProps( state ) {
+	const siteId = getSelectedSiteId( state );
+	const paymentMethods = getPaymentMethodsGroup( state, siteId, 'off-site' );
```

Comment:
This is only the offsite group.  In another component I will get the other groupings